app/graphql/schema.graphql

```diff
@@ -741,6 +746,51 @@ type Mutation {
   submitApplyingKyc(input: SubmitApplyingKycMutationInput!): SubmitApplyingKycMutationPayload
 }
 
+"""
+A float that must be non-negative. Used for currencies or prices.
+"""
+scalar PositiveFloat
+
+"""
+Pricefeed pair/symbols
+"""
+enum PricefeedPairEnum {
+  """
+  DAI to USD
+  """
+  DAI_USD
```

Comment:
I have made a change here: https://github.com/DigixGlobal/central_pricefeed/pull/18/commits/b7c6f44a150cfcb5b8b8371689e460a4ce688d74

This will only respond with the `xau` pairs, i.e. XAU to DAI, XAU to ETH, XAU to XBT, XAU to USD.

Reply:
Removed it